### PR TITLE
built and integrated navbar to all pages

### PR DESCRIPTION
**Pull request by gagandeep95**
I have added the navbar to all the pages When a user has not logged in, then only login and signup links displayed, After login, the user can see other information on the navbar
Also, I have added the empty content pages to booking and profile pages only to show the functionality.
Screenshots Below 

dashboard page
ipadversion
loginPage
moboversion

### CODE REVIEW

File: client/src/components/AuthMenu/AuthMenu.tsx [hunk 1]
```diff
@@ -40,6 +43,11 @@ const AuthMenu = (): JSX.Element => {
         }}
         getContentAnchorEl={null}
       >
+        <MenuItem>
```
Comment (rajivtitus) [inline client/src/components/AuthMenu/AuthMenu.tsx]:
```suggestion
        <MenuItem component={Link} to="/profile">
```

You can combine the MUI MenuItem with react-router's Link :)

File: client/src/components/Navbar/Navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+const Navbar = (): JSX.Element => {
+  const classes = useStyles();
+  const { loggedInUser } = useAuth();
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <Grid container className={classes.root}>
+      <Grid xs={5} md={3} className={classes.navbarBanner}>
+        <img src="https://res.cloudinary.com/dz8xmxmly/image/upload/v1636475121/fun_kickoff_project/logo_ddnmxs.png" />
```
Comment (rajivtitus) [inline client/src/components/Navbar/Navbar.tsx]: It is a good practice to add 'alt' attributes to images for accessibility purposes!

File: client/src/components/Navbar/Navbar.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      <Grid xs={12} md={9} item>
+        <Box
+          display={`${isOpen ? 'flex' : 'none'}`}
+          flexDirection={{ xs: 'column', sm: 'row' }}
+          className={classes.links}
+        >
+          {!loggedInUser ? (
+            <>
+              <Link to="/login" className={classes.authNavLinks}>
+                <Button color="primary" variant="outlined">
```
Comment (rajivtitus) [inline client/src/components/Navbar/Navbar.tsx]:
```suggestion
                <Button component={Link} to="/login" color="primary" variant="outlined">
```
Similar to the above case, the MUI Button can be combined with react-router's Link. Ditto throughout!

File: client/src/components/Navbar/Navbar.tsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+                  LOGIN
+                </Button>
+              </Link>
+              <Link to="/signup" className={classes.authNavLinks}>
+                <Button color="primary" variant="contained">
+                  SIGNUP
```
Comment (rajivtitus) [inline client/src/components/Navbar/Navbar.tsx]: Please avoid using CAPITALCASE for text, some screen readers interpret them as abbreviations, you can use a CSS class to transform text instead

Ditto throughout!

File: client/src/components/Navbar/Navbar.tsx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+          <MenuIcon />
+        </IconButton>
+      </Grid>
+      <Grid xs={12} md={9} item>
+        <Box
+          display={`${isOpen ? 'flex' : 'none'}`}
+          flexDirection={{ xs: 'column', sm: 'row' }}
+          className={classes.links}
+        >
+          {!loggedInUser ? (
```
Comment (rajivtitus) [inline client/src/components/Navbar/Navbar.tsx]: Optional: You can componentize both variants of the Navbar below for increase readability! 😄

File: client/src/pages/Bookings/Bookings.tsx [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+
+import Navbar from '../../components/Navbar/Navbar';
+function Bookings(): JSX.Element {
+  return (
+    <Grid container>
+      <Navbar />
```
Comment (rajivtitus) [inline client/src/pages/Bookings/Bookings.tsx:8]: What about rendering the Navbar outside the Switch component in the App.tsx to make it persist across the app? :)

**Review by rajivtitus — COMMENTED** [review]
Great work here! 💯 
- Good job adding mobile responsiveness
- I like the fact that you have created empty content pages 👍🏽 

Just some minor comments for you to address. Thanks!

File: client/src/components/AuthMenu/AuthMenu.tsx (hunk 1 above)
Comment (gagandeep95) [reply client/src/components/AuthMenu/AuthMenu.tsx]: Thanks for these suggestions

File: client/src/components/Navbar/Navbar.tsx (hunk 3 above)
Comment (gagandeep95) [reply client/src/components/Navbar/Navbar.tsx]: It just made my life easy! Thanks, Rajiv

File: client/src/components/Navbar/Navbar.tsx (hunk 4 above)
Comment (gagandeep95) [reply client/src/components/Navbar/Navbar.tsx]: Thanks, Rajiv, I will definitely remove the CAPTIALCASE, However, I was replicating the mock-up navbar.

File: client/src/components/Navbar/Navbar.tsx [hunk 7]
```diff
@@ -30,38 +32,7 @@ const Navbar = (): JSX.Element => {
...
-                  Bookings
-                </Link>
-              </Button>
-              <Button className={classes.navLinks} variant="text">
-                Messages*
-              </Button>
-              <AuthMenu />
-            </>
-          )}
+          {!loggedInUser ? <AuthNavLinks /> : <OtherNavLinks />}
```
Comment (rajivtitus) [inline client/src/components/Navbar/Navbar.tsx:35]: 👍🏽

File: client/src/components/Navbar/OtherNavLinks.tsx [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+        Become a sitter
+      </Button>
+      <Button className={classes.navLinks} variant="text">
+        Notifications
+      </Button>
+      <Button component={Link} to="/bookings" variant="text" className={classes.navLinks}>
+        Bookings
+      </Button>
+      <Button className={classes.navLinks} variant="text">
+        Messages*
```
Comment (rajivtitus) [inline client/src/components/Navbar/OtherNavLinks.tsx:20]:
```suggestion
        Messages
```
I'm guessing it's a typo haha but if you want to implement the green dot look into MUI badges :)

**Review by rajivtitus — APPROVED** [review]
LGTM! Great work making those changes 👍🏽

**Pull request merged**